_posts/2023-02-23-zennews-summarization.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: post
+author: "Baris Can Durak"
+title: "ZenNews: Generate summarized news on a schedule"
+description: "A tool powered by ZenML that can automate the summarization of news sources and save users time and effort while providing them with the information they need.."
+category: zenml
+tags: tooling zenml summarization news
+publish_date: February 23, 2023
+date: 2023-02-23T00:02:00Z
```

Comment:
```suggestion
publish_date: February 24, 2023
date: 2023-02-24T00:02:00Z
```
This (and the actual filename of the file) will need updating to whenever it gets published.